joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metafile client="installation" version="4.0">
 	<name>German (Switzerland)</name>
-	<version>4.0.0</version>
-	<creationDate>11.08.2021</creationDate>
+	<version>4.0.3v1</version>
```

Comment:
Should be 4.0.3
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L4

Reply:
```suggestion
	<version>4.0.3</version>
```